packages/ui-color-picker/src/ColorPreset/index.tsx

```diff
@@ -38,6 +38,7 @@ import { Text } from '@instructure/ui-text'
 import { Drilldown } from '@instructure/ui-drilldown'
 import type { DrilldownProps } from '@instructure/ui-drilldown'
 import { IconAddLine, IconCheckDarkSolid } from '@instructure/ui-icons'
+import { ScreenReaderContent } from '@instructure/ui-a11y-content'
```

Comment:
You need to remove this import because it is not used anymore. Also, you forgot to ignore the `button-name` rule in the a11y test where they break.